animators/src/main/java/com/revolut/recyclerkit/animations/BaseItemAnimator.kt

```diff
@@ -442,9 +308,8 @@ abstract class BaseItemAnimator(
         toX: Int, toY: Int
     ): Boolean {
 
-        if (oldHolder === newHolder && fromX == toX && fromY == toY) {
-            dispatchChangeFinished(newHolder, true)
-            return false
+        if (oldHolder === newHolder) {
+            return animateMove(oldHolder, fromX, fromY, toX, toY)
```

Comment:
This is an actual fix. animateMove instead of dispatchChangeFinished
Everything else was just  reordered to follow original DefaultItemAnimator order.